polaris/ocean/model/time.py

```diff
@@ -29,8 +31,8 @@ def get_time_interval_string(days=None, seconds=None):
 
     day_part = int(total / sec_per_day)
     sec_part = total - day_part * sec_per_day
-
+    sec_decimal = sec_part - np.floor(sec_part)
     # https://stackoverflow.com/a/1384565/7728169
     seconds_str = time.strftime('%H:%M:%S', time.gmtime(sec_part))
-    time_str = f'{day_part:04d}_{seconds_str}'
+    time_str = f'{day_part:04d}_{seconds_str}.{int(sec_decimal * 1e3)}'
```

Comment:
I don't think this will work quite right. I think the decimal part needs to be zero-padded:
```suggestion
    time_str = f'{day_part:04d}_{seconds_str}.{int(sec_decimal * 1e3):03d}'
```
Otherwise, 5 milliseconds will instead become 500 miliseconds.